package.json

```diff
@@ -0,0 +1,46 @@
+{
+  "name": "@gofynd/fp-signature",
+  "version": "0.0.1",
+  "description": "Fynd platform's signature logic",
+  "main": "./dist/cjs/index.js",
+  "module": "./dist/mjs/index.js",
+  "browser": "./dist/web/fp-signature.min.js",
+  "types": "./dist/types/index.d.ts",
+  "scripts": {
+    "build": "npm run clear && concurrently 'npm:build:cjs' 'npm:build:mjs' 'npm:build:types' 'npm:build:web'",
+    "build:cjs": "tsc -p tsconfig-cjs.json",
+    "build:mjs": "tsc -p tsconfig-mjs.json",
+    "build:web": "webpack",
+    "build:types": "tsc -p tsconfig-types.json",
+    "prepublishOnly": "npm run build",
+    "clear": "rimraf ./dist",
+    "test": "jest --coverage"
+  },
+  "repository": {
```

Comment:
Add keywords in package.json @bhargavprajapatiFynd

Reply:
Added keywords